### ISSUE TITLE
[Suggestion/Bug] rotator speeds do not affect turrets 

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#3349]
it would be interesting if the rotator/hinge speeds would also affect turrets so that we could make cool slow moving cinematic turrets by slowing them down (although turret speed should not be increasable as that would make them to OP when controlled by AI)  or if we could set on of the hinges/rotators to 0% speed so that we can have easy to make 1 axis turrets.

**Comment by zorozeenee** [comment]
I agree, ive run into this problem when trying to make one axis turrets
this is a good solution

**Comment by tsunamayo** [comment]
I want to introduce a speed mechanism that will depends on the mass of your turret. It could also use the velocity setting I guess... Cheers
